pyuavcan/transport/serial/__init__.py

```diff
@@ -64,32 +64,29 @@
 Total header size: 32 bytes (256 bits).
 
 The header is prepended before the frame payload; the resulting structure is
-encoded into its serialized form using the following packet format (influenced by HDLC, SLIP, POPCOP):
+encoded into its serialized form using the following packet format:
 
 +-------------------------+--------------+---------------+--------------------------------+-------------------------+
-| Frame delimiter **0x9E**|Escaped header|Escaped payload| Escaped CRC-32C of the payload | Frame delimiter **0x9E**|
+| Frame delimiter **0x00**|Escaped header|Escaped payload| Escaped CRC-32C of the payload | Frame delimiter **0x00**|
 +=========================+==============+===============+================================+=========================+
 | 1 byte                  | 32..64 bytes | >=0 bytes     | 4..8 bytes                     | 1 byte                  |
```

Comment:
With COBS the header and the CRC have fixed size now.

```suggestion
| 1 byte                  | 32 bytes     | >=0 bytes     | 4 bytes                        | 1 byte                  |
```